docs/pipelineruns.md

```diff
@@ -535,7 +571,7 @@ metadata:
   name: go-example-git
 spec:
   # […]
-  status: "PipelineRunCancelled"
```

Comment:
Hey @rafalbigaj lets keep this example as is with a note saying this `status` will be deprecated in V1, please use `Cancelled` instead. Also add a reference to the TEP in the doc. The rest of the changes to the doc looks good.

Reply:
Thanks @pritidesai for this suggestion!
I updated docs and added link to TEP.